### PR TITLE
Update free-programming-books-langs.md

### PR DESCRIPTION
**Pull request by kalpeshT101**
## What does this PR do?
Shopify hydrogen tutorial

### How do we know it's really free?
Official docs

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
please resolve conflicts.

**Comment by kalpeshT101** [comment]
@eshellman

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1223,6 +1224,11 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Fastify - Latest Documentation](https://www.fastify.io/docs/latest) (HTML)
 
 
+#### Hydrogen
+
+* [Build a Hydrogen storefront](https://shopify.dev/custom-storefronts/hydrogen/getting-started/tutorial) (HTML)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1229]: It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. I would put it in subjects/
Misc, unless you can find other books on application frameworks, commerce or something else.

Comment (kalpeshT101) [reply books/free-programming-books-langs.md:1229]: added another tutorial

Comment (eshellman) [reply books/free-programming-books-langs.md:1229]: that's more of a blog post. We''ll add the category and maybe reorganize later

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -1223,6 +1224,12 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
+* [Getting Started with Hydrogen](https://morioh.com/p/703736af058c)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1230]:
```suggestion
```

**Pull request merged**